app/controllers/comments_controller.rb

```diff
@@ -27,12 +27,12 @@ def set_user_and_post
     @post = @user.posts.find(params[:post_id])
   end
 
-  def store_referer
-    session[:referer] = request.referer
-  end
+  # def store_referer
+  #   session[:referer] = request.referer
+  # end
 
-  def redirect_back_or_default(default)
-    redirect_to(session[:referer] || default)
-    session.delete(:referer)
-  end
+  # def redirect_back_or_default(default)
+  #   redirect_to(session[:referer] || default)
+  #   session.delete(:referer)
+  # end
```

Comment:
[REQUIRED] Kindly remove unused code in order to make your code more professional